### PR TITLE
Harry item navbar

### PR DESCRIPTION
**Pull request by haoyuanli2023**
Added navbar for item and created test for it. Passed all test,coverage,mutation.
#5

### CODE REVIEW

**Comment by DistractedPrimate** [comment]
Can you resolve the merge conflicts with frontend/src/main/components/Nav/AppNavbar.js and
frontend/src/tests/components/Nav/AppNavbar.test.js.

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 1]
```diff
@@ -54,6 +54,13 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                     <NavDropdown.Item as={Link} to="/todos/create">Create Todo</NavDropdown.Item>
                   </NavDropdown>
                 )
+              }
+              {
+                hasRole(currentUser, "ROLE_USER") && (
```
Comment (DistractedPrimate) [inline frontend/src/main/components/Nav/AppNavbar.js]: code is repeated here

**Comment by haoyuanli2023** [comment]
> 
I have no idea what it is, I will ask in section.

**Review by roberto-garcia-0 — APPROVED** [review]
Ran the frontend and checked for the item navbar option. Ran tests and got full code and mutation coverage.

**Comment by anwu0203** [comment]
Can you resolve the merge conflicts? If you need help I can do it for you :)

**Comment by haoyuanli2023** [comment]
> Can you resolve the merge conflicts? If you need help I can do it for you :)
God! You know what, I resolved it, but when someone pushs new branch in. I need do it again.

**Comment by anwu0203** [comment]
ok let me resolve the conflicts and merge it really quickly

**Comment by haoyuanli2023** [comment]
> ok let me resolve the conflicts and merge it really quickly

Thank you so much Andy! Did you solve it?

**Comment by anwu0203** [comment]
yeah im waiting for the tests to pass and then im gonna merge it

**Comment by haoyuanli2023** [comment]
> 

Thank you I will resolve the conflict for my other PR

**Pull request merged**